### PR TITLE
DOC: Urge users to install into dedicated conda env

### LINKED ISSUE #33: Installation problems on Windows
Using the default command `pip install mnelab` on Windows I get:
```
Installing collected packages: PyQt5, mnelab
Could not install packages due to an EnvironmentError: [WinError 5] Access is denied: 'C:\\Users\\mmagn\\Continuum\\anaconda3\\Lib\\site-packages\\PyQt5\\Qt.pyd'
Consider using the `--user` option or check the permissions.
```
strangely, I get it even when I run it in command prompt with administrator privileges. When I run using `pip install mnelab --user` the script the `mnelab.exe` is put in a separate directory and does not do anything. When I then first activate my base anaconda env (it is my default python so I don't know why it makes any difference) and then do `pip install mnelab` - I get everything installed in one directory but `mnelab.exe` still does not work.
When I try to import mnelab in python I get:
```python-traceback
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\mmagn\Continuum\anaconda3\lib\site-packages\mnelab\__init__.py", line 1, in <module>
    from .mainwindow import MainWindow
  File "C:\Users\mmagn\Continuum\anaconda3\lib\site-packages\mnelab\mainwindow.py", line 7, in <module>
    from PyQt5.QtCore import (pyqtSlot, QStringListModel, QModelIndex, QSettings,
ModuleNotFoundError: No module named 'PyQt5.sip'
```

I might have been unlucky - I can test tomorrow on other Windows computers.

### PR DESCRIPTION
**Pull request by hoechenberger**
Fixes #33.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,14 +44,15 @@ pip install mnelab
 The `mnelab` command starts the application.
 
 #### Via conda
-An (unofficial, but regularly updated) conda package can be installed from [conda-forge](https://conda-forge.org/), ideally into
-its own dedicated environment:
+An (unofficial, but regularly updated) conda package can be installed from [conda-forge](https://conda-forge.org/).
+We **strongly** suggest to install `mnelab` into its own, dedicated environment to ensure smooth installation
```
Comment (cbrnr) [inline README.md]: I'd remove the comma between "own, dedicated"

**Comment by cbrnr** [comment]
Can you also include the `mnelab` line which starts the app in the pip section (and also change the last sentence accordingly)?

File: README.md (hunk 1 above)
Comment (hoechenberger) [reply README.md]: Done in 93d6870

**Comment by hoechenberger** [comment]
@cbrnr Done.

File: README.md [hunk 2]
```diff
@@ -40,18 +40,20 @@ MNELAB comes with the following features that are not (yet) available in MNE:
...
+mnelab
 ```
-The `mnelab` command starts the application.
+The `mnelab` command in the last line starts the application.
 
 #### Via conda
-An (unofficial, but regularly updated) conda package can be installed from [conda-forge](https://conda-forge.org/), ideally into
-its own dedicated environment:
+An (unofficial, but regularly updated) conda package can be installed from [conda-forge](https://conda-forge.org/).
+We **strongly** suggest to install `mnelab` into its own dedicated environment to ensure smooth installation
```
Comment (cbrnr) [inline README.md]: One last nitpick, I'd write MNELAB to refer to the app, `mnelab` is just the command to start it.

**Comment by cbrnr** [comment]
@mmagnuski let us know if this solves the problem or if you would like to have instructions on how to install MNELAB in conda's base env, i.e.
```
conda install numpy scipy matplotlib pyqt
pip install mne
pip install --no-deps mnelab
```

File: README.md (hunk 2 above)
Comment (hoechenberger) [reply README.md]: Done in 503e503

**Comment by mmagnuski** [comment]
I don't think instructions for installing into base conda env from conda-forge should be provided if it is expected to give problems - so the instructions are ok now. I would maybe add one sentence of explanation why it is not a good idea. But for this I can check with other computers installing to existing conda env. If it fails consistently then we should write that installing this package from conda-forge into existing env is expected to break the env. Even if this is a more general problem with conda-forge - it does not happen with all conda-forge packages so it still might be surprising for some conda users.
I can also test the option @cbrnr suggests.

**Comment by cbrnr** [comment]
OK, so I'll merge this, but if you find that we should add more info/instructions after your testing, please open a new issue.

**Comment by mmagnuski** [comment]
Sure!

**Pull request merged**